subprojects/docs/docs.gradle

```diff
@@ -115,69 +109,12 @@ outputs.docs = files(docsDir) {
     builtBy 'javadocAll', 'userguide', 'dslHtml', 'releaseNotes'
 }
 
-tasks.withType(CacheableAsciidoctorTask).configureEach {
-    backends = ['docbook']
-
-    separateOutputDirs = false
-    options doctype: 'book'
-    inputs.file asciidocSanitizeStylesheet withPropertyName "sanitizerStylesheet" withPathSensitivity PathSensitivity.NONE
-
-    extensions {
-        inlinemacro (name: "api") {
-            parent, target, attributes ->
-            def (className, method) = target.split('#', 2) as List
-            def content = "<apilink class=\"$className\""
-            if (method) {
-                // Add space after comma, because we can't have spaces in api: targets
-                // Restore Asciidoc's '&hellip;&zwsp;' replacement to the original '...'
-                content += " method=\"${method.replaceAll(/,\s*/, ', ').replaceAll(/\&#8230;\&#8203;/, '...')}\""
-            }
-            content += " />"
-            return content
-        }
-    }
-
-    attributes 'source-highlighter': 'prettify',
-               website: 'http://www.gradle.org',
-               javaApi: 'https://docs.oracle.com/javase/7/docs/api',
-               antManual: 'https://ant.apache.org/manual',
-               gradleVersion: version
```

Comment:
You changed these attributes from camel-case to hyphenated. Why? More importantly, I've noticed that some of the Asciidoc files still refer to these camel-case attributes — specifically `javaApi`.

Reply:
I copied over some config from the guides, which was hyphenated, and wanted to make it consistent. I don't have a strong preference here. I can keep it camelCase if you do.

Reply:
I favour consistency and working links :smile: I slightly prefer camel case, but I noticed that hyphenated fit with the builtin Asciidoctor attributes. So I'm OK with either. My main concern is that the user manual content matches the attributes defined in the build script.

Reply:
I have changed them all to camelCase for now, and will change all of the variable names as needed in all subsequent adoc PRs.